modules/configuration-annotation-processor/src/main/java/org/apache/ignite/internal/configuration/processor/ConfigurationProcessor.java

```diff
@@ -153,12 +156,16 @@ private boolean process0(RoundEnvironment roundEnvironment) {
             return false;
         }
 
+        var injectedValueValidator = new InjectedValueValidator(processingEnv);
```

Comment:
Note to reviewer: I'm going to refactor this part in a next PR to extract validators for all annotations, not just this one.